packages/react-native-sdk/docusaurus/docs/reactnative/05-advanced/02-push-notifications.mdx

```diff
@@ -413,123 +278,82 @@ import { navigationRef } from './src/utils/staticNavigationUtils';
 </NavigationContainer>
 ```
 
-### Setup the active call screen to process the incoming call from the push notification
+### Setup the push config
 
-The following is an example implementation of the screen that shows the active call. It is set up to accept the incoming call once a `call-id` for the incoming call is received.
+Once we have setup the methods to navigate the app from a static method we are ready to call the `StreamVideoRN.setPushConfig` method. Below is an example of how this method can be called,
 
-```ts
+```ts title="src/utils/setPushConfig.ts"
 import {
-  ActiveCall,
-  useActiveCall,
-  useIncomingCalls,
-  useRingCall,
+  StreamVideoClient,
+  StreamVideoRN,
 } from '@stream-io/video-react-native-sdk';
-
-export const CallScreen = ({ route, navigation }: Props) => {
-  const { callIdFromPushNotif } = route.params;
-  const activeCall = useActiveCall();
-  const [incomingCall] = useIncomingCalls();
-  const { answerCall } = useRingCall();
-
-  useEffect(() => {
-    // effect to answer call when incoming call is received from push notifications
-    if (!incomingCall?.call || !callIdFromPushNotif) {
-      return;
-    }
-    answerCall();
-    // remove the current call id to avoid rejoining when coming back to this screen
-    navigation.setParams({
-        callIdFromPushNotif: undefined,
-    });
-  }, [callIdFromPushNotif, incomingCall?.call]);
-
-  const onCallHungUpHandler = () => {
-    navigation.navigate('JoinCallScreen');
-  };
-
-  const onOpenCallParticipantsInfoViewHandler = () => {
-    navigation.navigate('CallParticipantsInfoScreen');
-  };
-
-  if (!activeCall) {
-    return <ActivityIndicator size={'large'} style={StyleSheet.absoluteFill} />;
-  }
-  return (
-    <ActiveCall
-      onCallHungUp={onCallHungUpHandler}
-      onOpenCallParticipantsInfoView={onOpenCallParticipantsInfoViewHandler}
-    />
-  );
+import { AndroidImportance } from '@notifee/react-native';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { STREAM_API_KEY } from '../../constants';
+import { StaticNavigate } from './staticNavigationUtils';
+
+export function setPushConfig() {
+  StreamVideoRN.setPushConfig({
+    ios: {
+      // add your push_provider_name for iOS that you have setup in Stream dashboard
+      pushProviderName: __DEV__ ? 'apn-video-staging' : 'apn-video-production',
+    },
+    android: {
+      // add your push_provider_name for Android that you have setup in Stream dashboard
+      pushProviderName: __DEV__ ? 'firebase-video-staging' : 'firebase-video-production',
+      // configure the notification channel to be used for incoming calls for Android.
+      incomingCallChannel: {
+        id: 'stream_incoming_call',
+        name: 'Incoming call notifications',
+        importance: AndroidImportance.HIGH,
```

Comment:
```suggestion
        // This is the advised importance of receiving incoming call notifications.
        // This will ensure that the notification will be on top of application
        importance: AndroidImportance.HIGH,
```